src/config/routes/index.js

```diff
@@ -0,0 +1,25 @@
+const jwt = require('koa-jwt');
+
+const config = require('config');
+
+const { urlToken, user } = require('infrastructure/middlewares');
```

Comment:
Let's try to avoid infastructure folder as long as we can :) That folder quickly becomes `mess` folder. In this particular case you can put middlewares right in the routes folder, near the `index.js`